### DISCUSSION TITLE
Clean Project!

### DISCUSSION THREAD

**Discussion by Mqxx** [Kira-NT/mc-publish#136] [General]
Hey I just looked at the source code of the project and I just want to say that it is one of the best open source typescript projects I have ever seen. Everything is carefully documented and clearly structured. You don't see something like this every day. Keep it up!

**Comment by Kira-NT** [comment]
Thanks a ton for the kind words! I don't really think this is "one of the best" TypeScript projects, as there are many other very, very cool ones out there. However, I genuinely appreciate the sentiment - it really means a lot ❤️
